spec/collections_publisher/publishing_parent_and_child_topic_spec.rb

```diff
@@ -0,0 +1,54 @@
+feature "Publishing a parent and child topic on Collections Publisher", collections_publisher: true do
+  include CollectionsPublisherHelpers
+
+  let(:topic_title) { title_with_timestamp }
+  let(:topic_slug) { slug_with_timestamp }
+  let(:link) { "/topic/" + topic_slug }
+
+  let(:child_slug) { slug_with_timestamp }
+  let(:child_title) { title_with_timestamp }
+
+  scenario "Publishing a parent and child topic" do
+    when_i_create_a_topic
+    when_i_publish_it
+    when_i_create_a_child_topic
+    when_i_publish_it
+    then_i_can_view_it_on_gov_uk
```

Comment:
I think this test could be rephrased a bit:

```
given_i_have_a_published_topic
when_i_add_a_child_topic
and_publish_it
then_i_can_view_it_on_gov_uk
```

I think since this test is only checking the child topic made it, it's also only really a test that a child topic can be published it's more of a "Publishing a child topic" test than "Publishing a parent and child topic"